src/applications/vaos/tests/mocks/v2.js

```diff
@@ -43,3 +44,28 @@ export function getVAOSRequestMock() {
     },
   };
 }
+
+/**
+ * Returns a stubbed var-resources parent site object from the VAOS service.
+ *
+ * @export
+ * @returns {VARParentSite} var-resources parent site object
+ */
+export function getVAOSParentSiteMock() {
```

Comment:
We could make this slightly easier to use by having an id parameter, and setting both ids, vista_site (the first three digits of it) and vast_parent to that value. That would make the tests that use it less verbose.

I'm trying to do this with more of our helpers.